analysis_options.yaml

```diff
@@ -147,7 +147,7 @@ linter:
     - prefer_typing_uninitialized_variables
     - prefer_void_to_null
     - provide_deprecation_message
-    - public_member_api_docs
+    # - public_member_api_docs
```

Comment:
Could you please undo this diff? I.e., just turn it back how it was?

Reply:
Done